db/migrate/20190723104817_create_contact_details.rb

```diff
@@ -0,0 +1,15 @@
+class CreateContactDetails < ActiveRecord::Migration[5.2]
+  def change
+    create_table :contact_details do |t|
+      t.string :phone_number
+      t.string :email_address
+      t.string :address
+
+      t.timestamps
+    end
+
+    remove_column :personal_details, :phone_number, :string
```

Comment:
Huh, cool trick including the field type, even on a remove - makes this is a reversible migration!